ethcore/src/engines/mod.rs

```diff
@@ -100,6 +135,7 @@ impl fmt::Display for EngineError {
 			FailedSystemCall(ref msg) => format!("Failed to make system call: {}", msg),
 			MalformedMessage(ref msg) => format!("Received malformed consensus message: {}", msg),
 			RequiresClient => format!("Call requires client but none registered"),
+			RequiresSigner => format!("Call requires client but none registered"),
```

Comment:
Copy + Paste from above?

```suggestion
			RequiresSigner => format!("Call requires signer but none registered"),
```

Reply:
Yes

Reply:
https://github.com/goerli/parity-goerli/pull/82/files